### ISSUE TITLE
add feature: forget password and design the UI for it

### ISSUE THREAD

**Issue by Parimalasri13** [Innovateninjas/Paws-frontend#298]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Parimalasri13** [comment]
hello @codewarnab , can you please assign adding of this new feature for me and I'm a GSSOC'24 contributor

**Comment by codewarnab** [comment]
yes we have an end point for that but we havnt added this yet it would be added  only after #270  closes .
we will assign you after its closed

**Comment by anshdeep0504** [comment]
Hey, allow me to work on this under GSSOC' 2024

**Comment by atharvarekhawar** [comment]
hey @codewarnab , I would like to work on this issue , can you assign this to me.

**Comment by codewarnab** [comment]
after #270 closes
